hoot-rnd/src/main/cpp/hoot/rnd/conflate/network/DebugNetworkMapCreator.cpp

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
@bmarchant I'll run some tests to try and figure out why these updates happen this way.  The years are based off git log information so it could be that this is the best we can do unless we want to come up with another technique and redo the copyright script.

@demmith and I will look further into it and make sure it's not a bug in the script.

This is how the copyrights have always been updated, but before when they were part of the release script the changes were never reviewed by an independent developer.